src/common/containers/app.js

```diff
@@ -5,7 +5,23 @@ import Helmet from 'react-helmet';
 import Header from '../components/header';
 import Footer from '../components/footer';
 
+import { fetchUser } from '../actions/user';
+
 export class App extends Component {
+  fetchData(props) {
+    if (props.auth.authenticated && !props.user.user && !props.user.isFetching) {
```

Comment:
App container fetches user data when user is authenticated.

Not an ideal solution as we need to wait for request to finish.
It's for example now visible in header, that we render page with Logout and Dashboard links, but user name shows up some time later (after we get data from GH).